addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -671,11 +671,27 @@
         </we-row>
         <!-- LINKS -->
         <we-row string="Links" class="o_design_tab_title">
+            <we-input data-customize-css-property=""
+                        data-css-property="font-size"
+                        data-selector-text="a:not(.btn), a.btn.btn-link"
+                        data-unit="px"/>
             <we-colorpicker data-customize-css-property=""
                             data-css-property="color"
                             data-selector-text="a:not(.btn), a.btn.btn-link"
                             data-no-transparency="true"
                             data-color-prefix="text-"/>
+        </we-row>
+        <we-row>
+            <we-fontfamilypicker data-selector-text="a:not(.btn), a.btn.btn-link"/>
+            <span></span> <!--Separate the select from the buttons (styling)-->
+            <we-button title="Bold" class="fa fa-fw fa-bold" data-no-preview="true" data-toggle="true"
+                        data-customize-css-property="bolder"
```

Comment:
Wait so this is a style change right? Now by default links are _bolder_ (rather than _bold_) and _italic_ (rather than not)?